jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -315,11 +451,13 @@ data class CodeModernizerSessionContext(
                     return null
                 }
             } catch (t: Throwable) {
-                LOG.error { t.message.toString() }
+                val error = "Unexpected error when executing bundled Maven"
```

Comment:
Same as my previous nit: I think we should specify which command was run everywhere that we LOG. It's not a big deal but just my opinion

Reply:
Just saw this - This is a safety catch if any unexpected error happen with setting up Runner, polling, or something unexpected. We do have specific error logs for when runner executes install command and when runner executes copy command. I noticed that I should've added the prefix for telemetry error though, so I'll include it in my followup PR.